qualtricssurvey/tests.py

```diff
@@ -36,6 +36,8 @@ def test_student_view(self):
         self.assertIn(url_frag, fragment.content)
         url_frag = '>" target="_blank">Begin Survey'
         self.assertIn(url_frag, fragment.content)
+        message = '<p>' + xblock.message + '</p>'
+        self.assertIn(message, fragment.content)
```

Comment:
There probably should be a specific test for the editable message. Like testing to make sure if you change the message it gets changed in the xBlock output?